### PR TITLE
Avoid disabling of hook on log

### PR DESCRIPTION
**Pull request by badeball**
This allows a user to log active handlers multiple times during a programs lifetime.

Related to https://github.com/mafintosh/why-is-node-running/issues/42.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,9 +23,8 @@ module.exports = whyIsNodeRunning
 function whyIsNodeRunning (logger) {
   if (!logger) logger = console
 
-  hook.disable()
   logger.error('There are %d handle(s) keeping the process running', active.size)
-  for (const o of active.values()) printStacks(o)
```
Comment (mafintosh) [inline index.js]: why is this change needed?

Comment (badeball) [reply index.js]: To be honest, I'm not really sure anymore. Given that `printStacks` is synchronous, I would assume it to make absolutely no difference because of the single-threaded nature, but I somehow distinctly remember that it did. I'm no longer working on the same project, so I unfortunately can't reproduce it.
